app/ui/src/main/java/com/fsck/k9/ui/settings/account/AccountSettingsActivity.kt

```diff
@@ -38,9 +52,31 @@ class AccountSettingsActivity : K9Activity(), OnPreferenceStartScreenCallback {
         loadAccount()
     }
 
+    override fun onItemSelected(parent: AdapterView<*>, view: View, position: Int, id: Long) {
+        val uuid = accountSpinner.selection.uuid
+
+        if (uuid == accountUuid)
+            return;
+
+        start(this, uuid)
```

Comment:
In my opinion starting a new Activity doesn't make for a nice user experience. Instead we could clear the fragment stack and add a fragment for the new account. 

However, being able to switch accounts requires quite a few additional changes. So this should probably be done in a separate PR.

Reply:
OK, leaving as-is for this PR